faq/logs.md

```diff
@@ -0,0 +1,22 @@
+# LOGS
+
+If you want to find log folder the simplest way is Help->Show Logs:
+
+![screenshot-2019-02-20-at-08 06 40](https://user-images.githubusercontent.com/2209596/53073064-4c144f80-34e7-11e9-97be-8ec1be2f7c7b.jpg)
+
+Also, you can find logs manually using next paths:
```

Comment:
"using next paths" -> "using the following paths"

Looks good otherwise! Merge it!